Client/Frontend/Browser/TabManager.swift

```diff
@@ -52,10 +52,13 @@ class TabManager {
         delegate?.didSelectedTabChange(tab, previous: previous)
     }
 
-    func addTab() -> Browser {
+    func addTab(requestURL: NSURL? = nil) -> Browser {
```

Comment:
I'd leave this change out of the patch. It might be worth putting in another PR, but it's not related to the issue here.